src/models/TestStationDAO.js

```diff
@@ -13,6 +13,22 @@ class TestStationDAOmock {
     return dbClient.scan({ TableName: this.tableName }).promise()
   }
 
+  getTestStationEmailByPNumber (testStationPNumber) {
```

Comment:
No directly related to a change you made, but this class is "TestStationDAOmock" - not correct.

Reply:
Done